### ISSUE TITLE
Document toolbar.button.useButtonTag property

### ISSUE THREAD

**Issue by joneff** [telerik/kendo-ui-core#4730]
The following property is missing from the API `toolbar.button.useButtonTag`. The property is important as it changes the rendering of toolbar buttons from anchor (`<a>`) to button (`<button>`) element.

As per customers suggestion, add it to typescripts definitions as well.

**Comment by veselints** [comment]
Plus write a documentation article with an example of how to apply the above configuration on different button types in the Toolbar. Explain why using buttons is important to make the Toolbar widget accessible.

**Comment by GaloisGirl** [comment]
The toolbar is not WCAG compliant, I submitter [a detailed ticket here](https://feedback.telerik.com/kendo-jquery-ui/1440061-wcag-accessibility-toolbar-is-not-wcag-compliant).
